src/liballoc/borrow.rs

```diff
@@ -479,3 +479,15 @@ impl<'a> AddAssign<Cow<'a, str>> for Cow<'a, str> {
         }
     }
 }
+
+#[stable(feature = "cow_str_add_assign_char", since = "1.41.0")]
+impl AddAssign<char> for Cow<'_, str> {
```

Comment:
Please add tests in src/liballoc/tests/cow_str.rs.

Reply:
Done